src/components/CountryDetails.jsx

```diff
@@ -0,0 +1,35 @@
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import SingleDetail from './SingleDetail';
+
+const CountryDetails = () => {
+  const { name } = useParams();
+  const { countries } = useSelector((store) => store.countries);
+
+  const country = countries
+    .slice(0, 50)
+    .find((country) => country.name.common === name);
+
+  const {
+    capital,
+    subregion,
+    population,
+    landlocked,
+    area,
+  } = country;
+
+  return (
+    <ul className="bg-primary">
+      <SingleDetail item="Capital" value={capital} />
+      <SingleDetail item="Subregion" value={subregion} />
+      <SingleDetail item="Population" value={population} />
+      <SingleDetail item="Area" value={area} />
+      <SingleDetail
+        item="Landlocked"
+        value={landlocked ? 'Yes' : 'No'}
+      />
+    </ul>
```

Comment:
Good job on implementing this view , I want just to mention the empty space below please make your page take the full height to match the design

![image (1)](https://github.com/martinkarugaba/react-redux-capstone/assets/59707859/e45a8360-c735-46fe-875d-ec958f8c7b70)